pipelines/fbc-release/fbc-release.yaml

```diff
@@ -326,25 +326,16 @@ spec:
           - name: revision
             value: $(params.taskGitRevision)
           - name: pathInRepo
-            value: tasks/create-internal-request/create-internal-request.yaml
+            value: tasks/publish-index-image/publish-index-image.yaml
       params:
-        - name: pipelineRunName
-          value: $(context.pipelineRun.name)
-        - name: inputDataFile
-          value: $(tasks.add-fbc-contribution-to-index-image.results.requestResultsFile)
-        - name: request
-          value: "publish-index-image-pipeline"
         - name: dataPath
           value: "$(context.pipelineRun.uid)/data.json"
-        - name: internalRequestParams
-          value:
-            - name: sourceIndex
-              value: "sharedRequestFile:json:.jsonBuildInfo"
-              jsonKey: ".index_image_resolved"
-            - name: targetIndex
-              value: $(tasks.update-ocp-tag.results.updated-targetIndex)
-            - name: retries
-              value: "0"
+        - name: sourceIndex
+          value: $(tasks.extract-index-image.results.indexImageResolved)
+        - name: targetIndex
+          value: $(tasks.add-fbc-contribution-to-index-image.results.requestTargetIndex)
+        - name: retries
```

Comment:
Two questions:

1. Why don't we want any retries? Wouldn't it be better to have something like `3`?
2. If we do want 0 retries, then why specify this when 0 is already the default for the task?

Reply:
I did not think about that as I was just moving the parameters around, but 3 sounds reasonable. I will change.

Reply:
changed.